tests/data_context/store/test_store_backends.py

```diff
@@ -292,13 +292,16 @@ def test_TupleS3StoreBackend_with_empty_prefixes():
 
 
 def test_TupleGCSStoreBackend():
-    pytest.importorskip("google-cloud-storage")
+    # pytest.importorskip("google-cloud-storage")
```

Comment:
since `test_TupleGCSStoreBackend()` passes now, can this line be commented out?

Reply:
I think so, yes. Especially since google-cloud-storage is in `requirements-dev.txt`.

If I'm wrong, this is easy to reverse later.